src/global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why do we 2 global.json, one in the src folder, one in the root folder.

Reply:
The global.jsons under src and test are for building with preview version.
About the third one under repos root folder: there is a recently added msbuild task in yaml, named "Ensure msbuild.exe can parse nuget.sln". It would fail if there is no global.json under repos root folder. (Failed build is here https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=2811147)
Shall I just keep the one under root folder?